<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestQtDnaSequencesDisplay</class>
 <widget class="QDialog" name="QtTestQtDnaSequencesDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTestQtDnaSequencesDisplay</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_sequence_2">
     <property name="text">
      <string>ACGTC</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="edit_name_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="edit_name_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_sequence_1">
     <property name="text">
      <string>ACGTA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="edit_name_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edit_sequence_4">
     <property name="text">
      <string>ACGTT</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edit_sequence_3">
     <property name="text">
      <string>ACGTG</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="edit_name_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QScrollArea" name="area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="area_content">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>660</width>
        <height>233</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="button_display">
     <property name="text">
      <string>&amp;Display</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="button_fasta">
     <property name="text">
      <string>&amp;Save as fasta</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
